webui/react/src/components/Page.tsx

```diff
@@ -0,0 +1,36 @@
+import { Typography } from 'antd';
+import React from 'react';
+
+import { WithCommonProps } from 'types';
+import { toHtmlId } from 'utils/string';
+
+import css from './Page.module.scss';
+
+const { Title } = Typography;
+
+interface Props {
+  title: string;
+  hideTitle?: boolean;
+}
+
+const defaultProps = {
+  hideTitle: false,
+};
+
+const Page: React.FC<WithCommonProps<Props>> = (props: WithCommonProps<Props>) => {
+  const classes = [ props.className, css.base ];
+
+  return (
+    <main className={classes.join(' ')} id={toHtmlId(props.title)}>
+      {/* DISCUSSION https://ant.design/components/page-header/ ?  */}
```

Comment:
thought: makes sense to use to use `PageHeader`. I think what you have is a good start. We can switch it up to `PageHeader` when we want to support more complex pages (thinking exp detail and trial detail, where we might want a back button)